pact-lang-api.js

```diff
@@ -329,8 +383,11 @@ module.exports = {
   simple: {
     exec: {
       createCommand: simpleExecCommand,
+      createLocalCommand: simpleLocalCommand,
       createPollRequest: simplePollRequestFromExec,
       createListenRequest: simpleListenRequestFromExec
     }
-  }
+  },
+  sendCommand: sendCommand,
```

Comment:
let's follow the pattern above and put these in some group, like `fetch` or `api` or something.

Reply:
Grouped the functions into Pact.fetch.send, Pact.fetch.local, Pact.fetch.poll, Pact.fetch.listen